hypothesis-python/src/hypothesis/internal/conjecture/shrinker.py

```diff
@@ -744,7 +740,7 @@ def try_lower_node_as_alternative(self, i, v):
         examples = self.examples_starting_at[i]
         for _ in range(3):
             random_attempt = self.engine.cached_test_function_ir(
-                [n.value for n in prefix], extend=len(nodes) * 2
+                [n.value for n in prefix], extend=len(nodes)
             )
```

Comment:
behavior change: same argument as above. `extend` is now in terms of number of choices, not the buffer. `* 2` -> `* 1` was again mostly arbitrary 🙂